HelpSource/Classes/Buffer.schelp

```diff
@@ -66,7 +66,7 @@ The number of channels for the Buffer. The default is 1.
 argument:: completionMessage
 A valid OSC message or a Function which will return one. A Function will be passed this Buffer as an argument when evaluated.
 argument:: bufnum
-An explicitly specified buffer number. Use of this argument is unnecessary and highly discouraged. The bufnum specified here will not be reserved, and may get allocated elsewhere in the future with unintended results. The best practice is to leave this unset and let the buffer number allocator choose the next available bufnum.
+An explicitly specified buffer number. Use of this argument is highly discouraged and unnecessary under normal usage. Providing a non-nil argument here will bypass the server's buffer allocator. Therefore, the buffer returned by this method may be allocated elsewhere in the future with unintended results.
```

Comment:
"highly unnecessary" is a bit too strong and I am not sure it is logical (can something be unnecessary to a certain degree?).

My suggestion would be:

Optionally, an explicit buffer number can be specified. This intentionally bypasses the server's buffer allocator, so if you haven't reserved this buffer number, you may get unexpected collisions. Use at your own risk.